test/cpp/interop/client.cc

```diff
@@ -81,6 +81,14 @@ DEFINE_string(default_service_account, "",
 DEFINE_string(service_account_key_file, "",
               "Path to service account json key file.");
 DEFINE_string(oauth_scope, "", "Scope for OAuth tokens.");
+DEFINE_bool(do_not_abort_on_transient_failures, false,
```

Comment:
Actually I  too do not like double-negative names like this one :( and I started off by naming this `abort_on_transient_failures` (with default value to `true`). However, I felt that the name `abort_on_transient_failures` (irrespective of its default value) gave the impression that abort was not called for non-transient failures - which is not true.
So I felt `do_not_abort_on_transient_failures` was a better name.. It does correctly seem to convey that normally abort is called on all failures and that we just do not want to call it on transient failures.